src/azure-cli/azure/cli/command_modules/role/_help.py

```diff
@@ -9,31 +9,33 @@
 
 helps['ad'] = """
 type: group
-short-summary: Manage Azure Active Directory Graph entities needed for Role Based Access Control
+short-summary: >-
+    Manage Microsoft Entra ID (formerly known as Azure Active Directory, Azure AD, AAD) entities needed for
+    Azure role-based access control (Azure RBAC) through Microsoft Graph API.
```

Comment:
```suggestion
    Manage Microsoft Entra ID entities (formerly known as Azure Active Directory entities) needed for
    Azure role-based access control (Azure RBAC) through Microsoft Graph API.
```

Reply:
@dbradish-microsoft, your advice on this?

Reply:
According to our discussion, I will keep its original form `Manage Microsoft Entra ID (formerly known as Azure Active Directory, Azure AD, AAD) entities`.